Content.Server/Magic/Components/SpellbookComponent.cs

```diff
@@ -29,7 +29,5 @@ public sealed class SpellbookComponent : Component
     public readonly Dictionary<string, int> InstantSpells = new();
 
     [DataField("learnTime")]
-    public float LearnTime = .75f;
-
-    public CancellationTokenSource? CancelToken;
+    public float LearnTime = 5.0f;
```

Comment:
Is this intended

Reply:
Fixed in [`809da63` (#13225)](https://github.com/space-wizards/space-station-14/pull/13225/commits/809da6395f63b2d4e6f1f9e7b44c378283c05119)